### ISSUE TITLE
The accuracy on voc is not as expected

### ISSUE THREAD

**Issue by QiZhangsama** [Vladkryvoruchko/PSPNet-Keras-tensorflow#65]
I used pspnet101_voc2012.h5 and pspnet101_voc2012.json to test VOC data，and it's mIOU is 82.51% after uploaded the results. But the real value is 85.4% in the paper. I want to know why.

This is my resutls:

Competition | Category | Score
-- | -- | --
Object Segmentation (comp6) | background | 95.69300
Object Segmentation (comp6) | aeroplane | 93.16350
Object Segmentation (comp6) | bicycle | 68.16070
Object Segmentation (comp6) | bird | 92.60880
Object Segmentation (comp6) | boat | 77.27700
Object Segmentation (comp6) | bottle | 82.46240
Object Segmentation (comp6) | bus | 91.86820
Object Segmentation (comp6) | car | 91.61700
Object Segmentation (comp6) | cat | 94.20940
Object Segmentation (comp6) | chair | 37.09920
Object Segmentation (comp6) | cow | 88.46340
Object Segmentation (comp6) | diningtable | 74.82890
Object Segmentation (comp6) | dog | 87.07190
Object Segmentation (comp6) | horse | 93.95220
Object Segmentation (comp6) | motorbike | 89.37420
Object Segmentation (comp6) | person | 90.35040
Object Segmentation (comp6) | pottedplant | 68.82600
Object Segmentation (comp6) | sheep | 90.44340
Object Segmentation (comp6) | sofa | 60.11600
Object Segmentation (comp6) | train | 85.58210
Object Segmentation (comp6) | tvmonitor | 79.56540
  | Object Segmentation (comp6) | 82.51110

**Comment by LeeTruth** [comment]
I‘m sorry to bother you, I have a small question to ask you.  Would you like to tell me which .py  file that you used to test your accuracy? I didn't find it here, I am grateful that you can reply to me.

**Comment by xiaowoniuli** [comment]
> 
> 
> I‘m sorry to bother you, I have a small question to ask you. Would you like to tell me which .py file that you used to test your accuracy? I didn't find it here, I am grateful that you can reply to me.

have you find it?

**Comment by LeeTruth** [comment]
> > I‘m sorry to bother you, I have a small question to ask you. Would you like to tell me which .py file that you used to test your accuracy? I didn't find it here, I am grateful that you can reply to me.
> 
> have you find it?

no, if i find it I will tell you, and if you find it ,please tell me ,thanks
